test/src/test/resources/fixtures/deep renames/deep-rename-with-argument-works.yml

```diff
@@ -0,0 +1,103 @@
+name: deep rename with argument works
+enabled:
+  current: false
+  nextgen: true
+overallSchema:
+  IssueService: |
+    service IssueService {
+      type Query {
+        issue: Issue
+      }
+      type Issue {
+        name(userId: ID!): String => renamed from detail.detailName
+      }
+    }
+underlyingSchema:
+  IssueService: |
+    type Issue {
+      detail: IssueDetails
+    }
+
+    type IssueDetails {
+      detailName(userId: ID!): String
+    }
+
+    type Query {
+      issue: Issue
+    }
+query: |
+  query {
+    issue {
+      name(userId: "USER-01")
+    }
+  }
+variables: {}
+serviceCalls:
+  current:
```

Comment:
do you need to declare `current` here if the test never runs for the current engine?

Reply:
you don't HAVE TO
I thought it's good to have it in case we want to fix it for the current gen. We already have the service call and the result so we don't need to tinker with the test, can jump straight to debugging